README.md

```diff
@@ -200,6 +200,58 @@ pipenv shell
 apt install lintian
 make test
 ```
+## Signing sources
+
+SecureDrop Workstation code spans across the following repositories:
+
+https://github.com/freedomofpress/securedrop-client
+https://github.com/freedomofpress/securedrop-debian-packaging
+https://github.com/freedomofpress/securedrop-proxy
+https://github.com/freedomofpress/securedrop-sdk
+https://github.com/freedomofpress/securedrop-workstation
+https://github.com/freedomofpress/qubes-template-securedrop-workstation
+
+
+### Release
+1. For each release, a tag will be signed in all the above repos.
```

Comment:
Ambiguously worded (perhaps intentionally so). What constitutes a "release" of the SecureDrop Workstation? We should be prepared to bump individual package versions, and release those versions, without bumping versions for _all_ packages. Assuming you agree, I suggest the following rewording:

> For each release of a package in the above repos, a signed tag will be pushed documenting the commit hash used to perform the build. 

We don't yet have a "parent" version of the SecureDrop Workstation, just various versions of the different components.

Reply:
Yes, ambiguous given that we haven't gone through the process, and I assume this is subject to change. The main idea here is that we want release to have signed tags for all it's components, and we need to (easily) know which specific commit hash has made its way into the release. 

We have two choices: 
* Create a tag for each version of the specific packages and store metadata in securedrop-packaging or elsewhere to know which package tag is included in which workstation release
*  Tag each securedrop release in the above repos. If `securedrop-proxy` is not updated, the tag for 0.1 and 0.2 point to the same commit hash. 

Tracking the tags in a build-release makefile target in securedrop-debian-packaging seems like the straightforward way.